BinaryObjectScanner/Protection/CopyX.cs

```diff
@@ -0,0 +1,182 @@
+                    // Samples: Redump ID 108150
+                    return "copy-X";
+                }
+            }
+
+            var strs = pex.GetFirstSectionStrings(".rdata");
+            if (strs != null)
+            {
+                // Samples: Redump ID 82475 (False positive, but probably catches original Emergency 2), would catch Redump ID 48393
+                if (strs.Any(s => s.Contains("optgraph.dll")))
+                    return "copy-X";
+            }
+
+            return null;
+        }
+
+        /// <inheritdoc/>
+#if NET20 || NET35
+        public Queue<string> CheckDirectoryPath(string path, IEnumerable<string>? files)
+#else
+        public ConcurrentQueue<string> CheckDirectoryPath(string path, IEnumerable<string>? files)
+#endif
+        {
+#if NET20 || NET35
+            var protections = new Queue<string>();
+#else
+            var protections = new ConcurrentQueue<string>();
+#endif
+
+            // Checks for Light
+            if (files == null)
+                return protections;
+
+            // Gets files in ZDAT*
+            // Excludes files with .x64 extension to avoid flagging Professional files.
+            var fileList = files.Where(f => !f.EndsWith(".x64", StringComparison.OrdinalIgnoreCase)).Where(f =>
+            {
+                f = f.Remove(0, path.Length);
+                f = f.TrimStart('/', '\\');
+                return f.StartsWith("ZDAT", StringComparison.OrdinalIgnoreCase);
+            }).OrderBy(f => f).ToList();
+            // Sorts list of files in ZDAT* so just the first file gets pulled, later ones have a chance of the ring
+            // intersecting the start of the file.
+            if (fileList.Count > 0)
+            {
+                try
+                {
+                    using var stream = File.OpenRead(fileList[0]);
+                    byte[] block = stream.ReadBytes(1024);
+
+                    var matchers = new List<ContentMatchSet>
+                    {
+                        // Checks if the file contains 0x00
+                        // Samples: Redump ID 81628
+                        new(Enumerable.Repeat<byte?>(0x00, 1024).ToArray(), "copy-X"),
+                        // Checks for whatever this data is.
+                        // Samples: Redump ID 84759, Redump ID 107929. Professional discs also have this data, hence the exclusion check.
+                        new(new byte?[]
+                        {
+                            0x02, 0xFE, 0x4A, 0x4F, 0x52, 0x4B, 0x1C, 0xE0, 0x79, 0x8C, 0x7F, 0x85, 0x04, 0x00, 0x46,
+                            0x46, 0x49, 0x46, 0x07, 0xF9, 0x9F, 0xA0, 0xA1, 0x9D, 0xDA, 0xB6, 0x2C, 0x2D, 0x2D, 0x2C,
+                            0xFF, 0x00, 0x6F, 0x6E, 0x71, 0x6A, 0xFC, 0x06, 0x64, 0x62, 0x65, 0x5F, 0xFB, 0x06, 0x31,
+                            0x31, 0x31, 0x31, 0x00, 0x00, 0x1D, 0x1D, 0x1F, 0x1D, 0xFE, 0xFD, 0x51, 0x57, 0x56, 0x51,
+                            0xFB, 0x06, 0x33, 0x34
```

Comment:
15 byte wide lines feels awkward. Please either use 8 or 16 bytes instead. This is a super incredible nitpick.